pagopa/io-app
# feat: [IOCOM-1412] Support for pagoPA logo above service logo on message cell

## Short description
This PR adds support for a fixed pagoPA logo on top of the service logo.
![Simulator Screenshot - iPhone 15 - 2024-06-19 at 10 28 49](https://github.com/pagopa/io-app/assets/5150343/46ef179a-fb21-4270-9b71-03f9230fa7bd)

## List of changes proposed in this pull request
- `DoubleAvatar` component that support the fixed pagoPA logo on top of the background one. 
  - **_Please note_** that this component is temporary and will be moved into the DS library in the future, when all developments are done
  - **_Please note_** that this component does not support a dynamic avatar yet. That will come in the future when there will also be API support
- `MessageListItem` picks between `Avatar` and `DoubleAvatar` based on an input value
- `WrappedMessageListItem`'s business logic enhanced to compute when to use the double avatar (for messages tagged with payment)

## How to test
Using the io-dev-api-server, generate some payment messages and check that their logo has the pagoPA one on top. Also check that other types of messages do not have the double logo
